code/game/objects/items/storage/backpack.dm

```diff
@@ -889,6 +894,25 @@ GLOBAL_LIST_EMPTY_TYPED(radio_packs, /obj/item/storage/backpack/marine/satchel/r
 	fuel_type = "utnapthal"
 	has_gamemode_skin = TRUE
 
+//this is to revert change for the backpack that are for flametrower usage.
+// so that they can use custom mix to refill those backpack
+/obj/item/storage/backpack/marine/engineerpack/flamethrower/afterattack(obj/target, mob/user, proximity)
+	if(!proximity)
+		return
+	if (!(istype(target, /obj/structure/reagent_dispensers/fueltank)))
```

Comment:
Can we then instead limit this to `istypestrict(target, /obj/structure/reagent_dispensers/fueltank) || istype(target, /obj/structure/reagent_dispensers/fueltank/custom)`?

Reply:
i don't know what you want me to do here the first or the second sugestion.

Reply:
The suggestion was to replace `if (!(istype(target, /obj/structure/reagent_dispensers/fueltank)))` with `if(!istypestrict(target, /obj/structure/reagent_dispensers/fueltank) && !istype(target, /obj/structure/reagent_dispensers/fueltank/custom))` but its moot if this item has a way to empty its contents.